modules/installation/pages/install-server.adoc

```diff
@@ -12,6 +12,50 @@ endif::[]
 
 
 
+{susemgr} Server is a {suse} base product from version 4.0.
+This section describes how to install {susemgr} Server from {sls} installation media.
+It assumes you have already registered the {susemgr} product with the {scc} and have a registration code.
+
+For information on registering with {scc}, or obtaining installation media, see xref:[installation-general-requirements]
+// I can't remember how we're supposed to do xrefs. Need to check with @jcayouette --LKB
+
+[NOTE]
+====
+If you are installing {susemgr} as a proof of concept, or this is a test installation, you can follow these instructions on a virtual machine rather than in production.
```

Comment:
see above where I said: This sentence isn't Simple English ;) And then I'm wondering why you do not want to install the server on a VM even in production if the VM is powerful enough.